myaws/ssm_parameter.go

```diff
@@ -91,3 +91,22 @@ func (client *Client) getSSMParametersPerChunk(names []*string, withDecryption b
 
 	return response.Parameters, nil
 }
+
```

Comment:
Add a GoDoc comment to the public method.

Reply:
Added.